app/views/hq/sponsors/index.html.haml

```diff
@@ -0,0 +1,37 @@
+- if @sponsors.empty?
```

Comment:
I think the benefit of positively saying "no sponsors found" is that it leaves no doubt in the mind of the user. If there's an empty table there then there still may be some doubt along the lines of "is this still waiting to load the sponsor records?" or "can it not find the sponsor records?". Having said that, I output a blank users table if there were no records.